# construct QUrl from local file.
# this handles special characters like #
# also it should be the same for Windows.
Index: xvst-2.5.1/src/tools.cpp
===================================================================
--- xvst-2.5.1.orig/src/tools.cpp	2014-04-13 15:15:55.000000000 +0200
+++ xvst-2.5.1/src/tools.cpp	2014-04-13 15:25:47.599177107 +0200
@@ -527,11 +527,7 @@
 
 bool openFile(QString filePath)
 {
-#ifdef Q_OS_WIN32
-	return QDesktopServices::openUrl("file:///" + validPath(filePath)); //filePath);
-#else
-	return QDesktopServices::openUrl("file://" + validPath(filePath)); //filePath);
-#endif
+	return QDesktopServices::openUrl(QUrl::fromLocalFile(validPath(filePath)));
 }
 
 bool openDirectory(QString directoryPath)
